data/strings/types_strings.txt

```diff
@@ -18938,7 +18938,7 @@
     fa = ﺮﯾﻮﮐ
     fi = Aavikko
     fr = Désert
-    he = רבָּדמִ
+    he = רבָּדמִ
```

Comment:
NFD, unreadable

Reply:
What is "unreadable"? What's the correct version?

Reply:
מדבר

Reply:
Sorry, I didn't want to get into correcting everything, so I avoided correcting anything. Might have gone a bit overboard.

Some of the oldest automatic translations are written backwards, because the translation script used to have a feature enabled to make it readable in a terminal. There's probably a lot more of them still, and at least one or two were fixed elsewhere in this PR. I can't read Arabic but I'll bet there's a bunch of these in Arabic too.